web/src/beta/components/Resizable/hooks.ts

```diff
@@ -112,6 +122,10 @@ export default (
     }),
     [onResizeStart],
   );
+  const handleResetSize = useCallback(() => {
+    setMinimized(false);
+    setSize(minSize);
```

Comment:
Should be initialSize, not minSize here

Reply:
I set it to be minSize based on your first requirement asit is applicable now after last modification

Reply:
done